docs/index.bs

```diff
@@ -136,155 +90,193 @@ spec: rfc7230; urlPrefix: https://tools.ietf.org/html/rfc7230
 spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
     type: dfn
         text: Vary; url: section-7.1.4
+
+spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-referrer-policy/
+    type: dfn
+        text: Parse a referrer policy from a Referrer-Policy header; url: parse-referrer-policy-from-header
+</pre>
+
+<pre class="biblio">
+{
+  "unsanctioned-tracking": {
+    "href": "https://www.w3.org/2001/tag/doc/unsanctioned-tracking/",
+    "title": "Unsanctioned Web Tracking",
+    "date": "17 July 2015",
+    "status": "Finding of the W3C TAG",
+    "publisher": "W3C TAG"
+  }
+}
 </pre>
 
 <section>
-  <h2 id='motivations'>Motivations</h2>
+  <h2 id="motivations">Motivations</h2>
 
-  <em>This section is non-normative.</em>
+  *This section is non-normative.*
 
-  <p>Web Applications traditionally assume that the network is reachable. This assumption pervades the platform. HTML documents are loaded over HTTP and traditionally fetch all of their sub-resources via subsequent HTTP requests. This places web content at a disadvantage versus other technology stacks.</p>
+  Web Applications traditionally assume that the network is reachable. This assumption pervades the platform. HTML documents are loaded over HTTP and traditionally fetch all of their sub-resources via subsequent HTTP requests. This places web content at a disadvantage versus other technology stacks.
 
-  <p>The <a href="#dfn-service-worker">service worker</a> is designed first to redress this balance by providing a Web Worker context, which can be started by a runtime when navigations are about to occur. This event-driven worker is registered against an origin and a path (or pattern), meaning it can be consulted when navigations occur to that location. Events that correspond to network requests are dispatched to the worker and the responses generated by the worker may override default network stack behavior. This puts the <a href="#dfn-service-worker">service worker</a>, conceptually, between the network and a document renderer, allowing the <a href="#dfn-service-worker">service worker</a> to provide content for documents, even while offline.</p>
+  The <a for="/">service worker</a> is designed first to redress this balance by providing a Web Worker context, which can be started by a runtime when navigations are about to occur. This event-driven worker is registered against an origin and a path (or pattern), meaning it can be consulted when navigations occur to that location. Events that correspond to network requests are dispatched to the worker and the responses generated by the worker may override default network stack behavior. This puts the <a for="/">service worker</a>, conceptually, between the network and a document renderer, allowing the <a for="/">service worker</a> to provide content for documents, even while offline.
```

Comment:
I like it! `[=/service worker=]`, `[=ServiceWorkerGlobalScope/service worker=]` work fine and are shorter to type.